src/langcheck/metrics/eval_clients/_prometheus.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from typing import Iterable
+
+import torch
+from transformers import LlamaTokenizer, MistralForCausalLM
+
+from langcheck.utils.progess_bar import tqdm_wrapper
+
+from ._base import EvalClient
+
+
+class PrometheusEvalClient(EvalClient):
+    '''EvalClient defined for the Prometheus 2 model.
+    Presented in `"Prometheus 2: An Open Source Language Model Specialized
+    in Evaluating Other Language Models" <https://arxiv.org/abs/2405.01535>`
+    '''
+
+    def __init__(self, torch_dtype: torch.dtype, device: str):
```

Comment:
I agree. But we need to use `torch.bfloat16` since it's [the default setting of Prometheus.](https://huggingface.co/prometheus-eval/prometheus-7b-v2.0/blob/main/config.json)

Reply:
Ah sorry it was just a typo!!

I was suggesting that setting these parameters as the default values